projects/ion/src/lib/alert/alert.component.scss

```
@@ -47,11 +58,33 @@
   @include alert-style($negative-1, $negative-6);
 }
 
+.with-description {
+  border: none !important;
+  background-color: $neutral-2;
+  padding: 16px 24px !important;
```

sorry, forgot to sugest the sass functions.
```suggestion
  padding: spacing(2) spacing(3) !important;
```